include/lsst/geom/Box.h

```
@@ -163,34 +167,34 @@ class Box2I {
      *  Return the size of the box in pixels.
      */
     //@{
-    Extent2I const getDimensions() const { return _dimensions; }
-    int getWidth() const { return _dimensions.getX(); }
-    int getHeight() const { return _dimensions.getY(); }
-    int getArea() const { return getWidth() * getHeight(); }
+    Extent2I const getDimensions() const noexcept { return _dimensions; }
+    int getWidth() const noexcept { return _dimensions.getX(); }
+    int getHeight() const noexcept { return _dimensions.getY(); }
+    int getArea() const noexcept { return getWidth() * getHeight(); }
```

What happens if the area overflows an `int`? I suspect that should raise an exception (even if it does not), so I suggest not adding `noexcept` here.